data/resources.json

```diff
@@ -322,12 +352,17 @@
         },
         {
           "value": "axelartest-grpc.quantnode.tech:9090"
-        },
+        }
+      ]
+    },
+    {
+      "name": "gRPC-Web endpoints",
+      "value": [
         {
-          "value": "axelartest-grpcweb.quantnode.tech:9091"
+          "value": "https://axelartest-grpcweb.quantnode.tech"
         },
         {
-          "value": "https://axelartest-grpcweb.quantnode.tech"
+          "value": "https://axelartest-grpcweb.quantnode.tech:9091"
```

Comment:
Is this duplication correct?

Reply:
i did not remove the fields in the previous document. (`..grpcweb.quantnode.tech:9091` and `grpcweb.quantnode.tech`.) but i think we can rm this one (at L365)